### DISCUSSION TITLE
npm @ory/hydra-client modify subject when acceptin consent request

### DISCUSSION THREAD

**Discussion by pommelinho** [ory/hydra#2365] [Q&A] [answered]
I am using npm package @ory/hydra-client version 1.9.2 in my node server.
I am calling method `acceptLoginRequest` where i can pass a `AcceptLoginRequest` object. Here i can set the **subject** which is then set  as the `sub` claim in the access_token. 
However i want  to also modify the **subject** when user is providing consent and calling `acceptConsentRequest`.  But i do not see any possibility to do this in the `AcceptConsentRequest` type.
Is it somehow possible to adapt the subject in consent phase?

**Comment by vinckr** [comment, marked as answer]
Sorry, missed your question here. 
Are you still stuck on this or could you give any more info on the use case you are trying to solve? 
Right from the bat I dont think it is possible, you can set it in login, why do you want to set it again?
`you may also override this value with your own obfuscated sub value by setting force_subject_identifier when accepting the login challenge in your user login app.`

**Reply by pommelinho** [reply]
Hey!
Thanks for your help. 
I do not need this functionality anymore anyway ;-).
So all is good ;-)

**Reply by honzapatCZ** [reply]
I would actuqlly see this usefull in like switch users from the consent
